core/assets/bundles/bundle_vi.properties

```diff
@@ -566,41 +566,41 @@ requirement.core = Phá hủy căn cứ địch ở {0}
 requirement.research = Nghiên cứu {0}
 requirement.produce = Sản lượng {0}
 requirement.capture = Chiếm {0}
-requirement.onplanet = Control Sector On {0}
-requirement.onsector = Land On Sector: {0}
+requirement.onplanet = Kiểm soát khu vực {0}
+requirement.onsector = Đáp trên khu vực:t{0}
 launch.text = Phóng
 research.multiplayer = Chỉ máy chủ mới có thể nghiên cứu các mục.
 map.multiplayer = Chỉ máy chủ mới có thể xem các khu vực.
 uncover = Khám phá
 configure = Tùy chỉnh vật phẩm
-objective.research = [accent]Research:\n[]{0}[lightgray]{1}
-objective.produce = [accent]Obtain:\n[]{0}[lightgray]{1}
-objective.destroyblock = [accent]Destroy:\n[]{0}[lightgray]{1}
-objective.destroyblocks = [accent]Destroy: [lightgray]{0}[white]/{1}\n{2}[lightgray]{3}
-objective.item = [accent]Obtain: [][lightgray]{0}[]/{1}\n{2}[lightgray]{3}
-objective.coreitem = [accent]Move into Core:\n[][lightgray]{0}[]/{1}\n{2}[lightgray]{3}
-objective.build = [accent]Build: [][lightgray]{0}[]x\n{1}[lightgray]{2}
-objective.buildunit = [accent]Build Unit: [][lightgray]{0}[]x\n{1}[lightgray]{2}
-objective.destroyunits = [accent]Destroy: [][lightgray]{0}[]x Units
-objective.enemiesapproaching = [accent]Enemies approaching in [lightgray]{0}[]
-objective.destroycore = [accent]Destroy Enemy Core
-objective.command = [accent]Command Units
-objective.nuclearlaunch = [accent]⚠ Nuclear launch detected: [lightgray]{0}
-announce.nuclearstrike = [red]⚠ NUCLEAR STRIKE INBOUND ⚠
+objective.research = [accent]Nghiên cứu:\n[]{0}[lightgray]{1}
+objective.produce = [accent]Sản xuất:\n[]{0}[lightgray]{1}
+objective.destroyblock = [accent]Phá hủy:\n[]{0}[lightgray]{1}
+objective.destroyblocks = [accent]Phá hủy: [lightgray]{0}[white]/{1}\n{2}[lightgray]{3}
+objective.item = [accent]Có được: [][lightgray]{0}[]/{1}\n{2}[lightgray]{3}
+objective.coreitem = [accent]Cho vào căn cứ:\n[][lightgray]{0}[]/{1}\n{2}[lightgray]{3}
+objective.build = [accent]Xây: [][lightgray]{0}[]x\n{1}[lightgray]{2}
+objective.buildunit = [accent]Tạo ra quân lính: [][lightgray]{0}[]x\n{1}[lightgray]{2}
+objective.destroyunits = [accent]Tiêu diệt: [][lightgray]{0}[]x Units
```

Comment:
"Units" -> "kẻ địch"

Reply:
> 

you see ? Its destroy unit, not build unit bruh

Reply:
I not see that, sr